### DISCUSSION TITLE
Recommendations for Linux fontconfig

### DISCUSSION THREAD

**Discussion by asteroidcow** [fabrizioschiavi/pragmatapro#198] [Show and tell]
Hi, thanks for making this great font. I recently bought this and am interested in figuring out the best anti-aliasing / hinting / lcdfilter / autohint / bitmap configuration for this font on Linux. Is there a particular setting which you'd recommend with recent versions of fontconfig (e.g. 2.13)?

Also, thanks for sharing the emacs config for ligatures!

**Comment by asteroidcow** [comment]
Also, apologies if Github issues isn't the right forum for this question! Please feel free to close if that's the case.

**Comment by asteroidcow** [comment]
Maybe it'll help someone -- I found `hintnone` to produce the best looking text with this font. I wrote a fontconfig rule for this:
```
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
<fontconfig>
  <its:rules xmlns:its="http://www.w3.org/2005/11/its" version="1.0">
    <its:translateRule translate="no" selector="/fontconfig/*[not(self::description)]"/>
  </its:rules>
  <description>Disable hinting for PragmataPro</description>
  <match target="font">
    <test name="family" compare="contains" ignore-blanks="true">
      <string>PragmataPro</string>
    </test>
    <edit mode="assign" name="antialias">
      <bool>true</bool>
    </edit>
    <edit mode="assign" name="embeddedbitmap">
      <bool>false</bool>
    </edit>
    <edit mode="assign" name="hinting">
      <bool>false</bool>
    </edit>
    <edit mode="assign" name="hintstyle">
      <const>hintnone</const>
    </edit>
    <edit mode="assign" name="lcdfilter">
      <const>lcddefault</const>
    </edit>
    <edit mode="assign" name="rgba">
      <const>rgb</const>
    </edit>
  </match>
</fontconfig>
```

**Reply by lobre** [reply]
Hey, I am not sure I understand the point of removing the hints from this font while it is one of the key strengths of PragmataPro. What is the reasoning behind this?
